hw2/05-form.js

```diff
@@ -1 +1,13 @@
-// Add your code here
+document.addEventListener("DOMContentLoaded", () => {
+    const form = document.getElementById('userForm');
+    const modal_element = document.getElementById('responseModal');
+
+    const handleSubmit = (event) => {
+        //Prevent page reload on submit
+        event.preventDefault();
+
+        console.log('Event hit');
+    }
+
+    form.addEventListener('submit', handleSubmit);
+})
```

Comment:
Maybe the reason for you getting "bootstrap is undefined" is because in html file you have used attributes of bootstrap 4 and not 5, and you have included the bootstrap 5 as href. 

It is better to maintain same bootstrap version i.e bootsrap 5 attributes across the code.

Reply:
That was not the only reason, as I had tried that before, but this is good to know regardless so thank you.

Reply:
Good to hear that you have resolved the issue with "the boostrap is undefined".